lib/templates/imageFunction.js

```diff
@@ -0,0 +1,26 @@
+const jimp = require("jimp");
+
+// Function used to mimic next/image and sharp
+exports.handler = async (event) => {
+  const { url, w = 500, q = 75 } = event.queryStringParameters;
```

Comment:
not sure about these defaults but i think it's fine since next_image should almost always provide `w`

Reply:
we probably want some reasonable default just in case so we don't end up trying to process 6MB images at full res or whatever, but I think you're right that it _should_ always be set